package.json

```diff
@@ -25,6 +25,7 @@
 		"build:feature-config": "php bin/generate-feature-config.php",
 		"build:packages": "node ./bin/packages/build.js",
 		"build:release": "./bin/build-plugin-zip.sh",
+		"build:release-dryrun": "DRY_RUN=1 ./bin/build-plugin-zip.sh",
```

Comment:
This isn't overly necessary.

Reply:
Yeah I agree lets remove this additional command.

Reply:
It would be great if we could add a README instruction for doing the dry run in place of adding this command. Its a nit pick but it'd be neat if dry run was the default and you had to opt in to a real release instead. That way you're kinda encouraged to run a dry run, check it, then run the real release when you're done.